index.html

```diff
@@ -5,7 +5,7 @@
   <meta property="og:title" content="Don't ask to ask, just ask" />
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://dontasktoask.com" />
-  <meta property="og:description" content="Don't ask &quot;Any Java experts around?&quot;, but rather ask &quot;How do I do [problem] with Java and [other relevant info]?.&quot;" />
+  <meta property="og:description" content="Ask your questions directly instead of asking to ask. />
```

Comment:
What about something in line of `Hey, you! Don't poke around to see someone, ask your question in full right away. Thanks!`? In my opinion that seems way better than going full passive-agressive with "ask directly, don't ask to ask", however I'd love to hear what others think of that. I'm going to leave this as a comment for now.

Reply:
@revi what do you think?